lib/rubocop/cop/betterment/utils/parser.rb

```diff
@@ -88,16 +88,16 @@ def self.get_extracted_parameters(node, param_aliases: []) # rubocop:disable Met
             return [] unless node.send_type?
 
             parameter_names = []
-            param_aliases << :params
+            aliases = param_aliases | %i(params)
```

Comment:
__comment:__ Here is where the arguments were being mutated.

IMHO unless a method's *sole reason* for existing is to mutate arguments, and it's clearly a part of the documented interface, it should assume all arguments are immutable.

And even then, I think mutating arguments is questionable and should only be a last resort. Often there are many other options that are cleaner and simpler.